pop/POPAnimation.mm

```diff
@@ -270,3 +270,35 @@ - (id)pop_animationForKey:(NSString *)key
 }
 
 @end
+
+@implementation POPAnimation (NSCopying)
+
+- (instancetype)copyWithZone:(NSZone *)zone
+{
+  /*
+   * Must use [self class] instead of POPAnimation so that subclasses can call this via super.
+   * Even though POPAnimation and POPPropertyAnimation throw exceptions on init,
+   * it's safe to call it since you can only copy objects that have been successfully created.
+   */
+  POPAnimation *copy = [[[self class] allocWithZone:zone] init];
+  
+  if (copy) {
+    
```

Comment:
Are you saying the newlines right under the if? Happy to fix that, but not exactly sure :P